package.json

```diff
@@ -43,6 +43,14 @@
         "react": "^18.2",
         "react-dom": "^18.2"
     },
+    "husky": {
+        "hooks": {
+            "pre-commit": "lint-staged"
+        }
+    },
+    "lint-staged": {
+        "*.{js,jsx,ts,tsx,json}": "eslint --fix"
```

Comment:
I'd prefer it not to use `--fix`, I dislike it changing code without reviewing it.   Also it probably make more sense to run `yarn pf` since that also type checks the code as well